src/webpack/plugin/siteLoaderUtils.js

```diff
@@ -22,6 +25,21 @@ const getMdxLoader = (publicPath) => ({
         }
         return url.href;
       }],
+      [rehypeUrlInspector, {
+        inspectEach: (url) => {
+          if ((url.url.includes('https') || url.url.includes('http')) && !url.file.history[0].includes('node_modules')) {
```

Comment:
I think we should go ahead and check url's in node module files. It's good to know when you have broken urls regardless of where they come from.

Reply:
Removed it here https://github.com/cerner/terra-dev-site/pull/330/commits/cef108bcdeb40393bce44105c8f70ce21d06c956.